check50/__main__.py

```diff
@@ -266,8 +289,20 @@ def main():
                         metavar="FILE",
                         help=_("file to write output to"))
     parser.add_argument("-v", "--verbose",
+                        action="store",
+                        nargs="?",
+                        default="",
+                        const="info",
+                        choices=["info", "debug"],
+                        help=_("sets the verbosity level and implies --log."
+                               ' "info" displays the full tracebacks of errors and shows all commands run.'
+                               ' "debug" adds the output of all command run.'))
```

Comment:
Well, we also have `--log` that specifies whether we show the log of the individual checks. I think it would be rather confusing to rename `--verbose` to `--log-level` as the log level does not impact the `--log` we keep for every check.

Reply:
Hm I think `--log` is very generic tbh. It's not clear at all that this specifically enables logging in checks. But let's just go with this for now to move things forward and perhaps change in the future.